www/js/core/app.js

```diff
@@ -14,6 +14,7 @@ App = {
   urlLogout: function(){ return RmSetting.endPoint() + "/users/sign_out.json?auth_token=" },
   urlField: function(){ return RmSetting.endPoint() + "/v1/collections/" },
   urlSite: function(){ return RmSetting.endPoint() + "/v1/collections/" },
+  urlSiteThreshold: function(){ return RmSetting.endPoint() + "/v1/sites/alerted_to_reporters" },
   log: function (text, data) {
     if (App.DEBUG)
       console.log(text, data);
```

Comment:
should be removed debugging code

Reply:
We created function to console.log in app in the way that we can enable and disable Debug mode in production.